test/channels.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const DC = require('diagnostics_channel');
+
+const Code = require('@hapi/code');
+const Lab = require('@hapi/lab');
+
+const Hapi = require('..');
+
+const { describe, it } = exports.lab = Lab.script();
+const expect = Code.expect;
+
+describe('DiagnosticChannel', () => {
+
+    describe('onServerChannel', () => {
```

Comment:
It seems clearer to name it after the exposed channel name, eg. `hapi.onServer`.